test/test_type_promotion.py

```diff
@@ -2,11 +2,16 @@
 
 import torch
 import itertools
+import unittest
 
-from torch.testing._internal.common_utils import TestCase, run_tests, load_tests
+from torch.testing._internal.common_utils import TestCase, run_tests, load_tests, \
+    TEST_NUMPY
```

Comment:
It's probably fine to put `TEST_NUMPY` to the previous line. Or we can use Python 3 style:

    from torch.testing._internal.common_utils import (TestCase, run_tests, load_tests,
        TEST_NUMPY)

Reply:
True. The test files aren't consistent in the style they use for imports.